02_Lab2_NimishMudhale/bubble_sort.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(s []int) []int {
+	n := len(s)
+	sorted := false
+	for !sorted {
+		swapped := false
```

Comment:
It looks like `swapped` and `sorted` are basically opposites of each other. Remove one.

Reply:
Done